.github/workflows/image_rs_build.yml

```diff
@@ -1,9 +1,20 @@
 name: image-rs build
-on: [push, pull_request, create]
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'image-rs/**'
+      - '.github/image_rs_build.yml'
```

Comment:
Hey @katexochen , I'm not strong with github workflows, but should this be `.github/workflow/image_rs_build.yml`? Similar thing for `.github/workflow/ocicrypt_rs_build.yml`

Reply:
Yes, thanks for catching this. :)